fleetspeak/src/server/db/store.go

```diff
@@ -125,6 +125,14 @@ type MessageStore interface {
 	// DeletePendingMessages removes all pending messages for given clients.
 	DeletePendingMessages(ctx context.Context, ids []common.ClientID) error
 
+	// GetPendingMessageCount returns the number of pending messages for the given clients.
+	GetPendingMessageCount(ctx context.Context, ids []common.ClientID) (uint64, error)
+
+	// GetPendingMessages returns all pending messages for given clients.
```

Comment:
I'd say `returns pending messages` instead of `returns all pending messages` here, as you can apply `offset` and `limit` and get a range of messages. The description of `offset` and `limit` arguments below already mentions that you can get all messages if you set them to 0.

Reply:
Done.